src/main/scala/mesosphere/marathon/core/appinfo/impl/AppInfoBaseData.scala

```diff
@@ -48,6 +50,11 @@ class AppInfoBaseData(
     }
   }
 
+  lazy val tasksMap: Future[TaskTracker.AppDataMap] = {
```

Comment:
wouldn't tasksMapFuture be more consistent with the rest of the names in this class?